docs/articles_en/openvino-workflow-generative/inference-with-genai/inference-with-genai-on-npu.rst

```diff
@@ -416,66 +473,68 @@ model weights is encrypted.
                ov::genai::LLMPipeline pipe(model_path, "NPU", pipeline_config);
 
 
-Disable memory allocation
-+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-In case of execution failures, either silent or with errors, try to update the NPU driver to
-`32.0.100.3104 or newer <https://www.intel.com/content/www/us/en/download/794734/intel-npu-driver-windows.html>`__.
-If the update is not possible, set the ``DISABLE_OPENVINO_GENAI_NPU_L0``
-environment variable to disable NPU memory allocation, which might be supported
-only on newer drivers for Intel Core Ultra 200V processors.
-
-Set the environment variable in a terminal:
-
-.. tab-set::
+Whisper Inference on NPU
+###############################################################################################
 
-   .. tab-item:: Linux
-      :sync: linux
+OpenAI Whisper support (for
```

Comment:
good and maybe let's elaborate: why would I as a user need to know that? previous OV version won't work with these models? the sentence feels a bit in the air here with no end result :)

Reply:
Reworked this section a bit